doc/frame_lifetime.md

```diff
@@ -31,6 +31,20 @@ As a side-note, `rs2::device::stop` will block until all pending callbacks retur
 * Except some initial stabilization period, librealsense ensures no heap allocations are being made when using frame callbacks. (This also applies to `rs2::frame_queue` but not to `rs2::syncer` primitive)
 * If you are not releasing `rs2::frame` objects in less then the `1000 / fps` milliseconds, you will likely encounter frame drops. These events will be visible in the log, if you decrease the severity to DEBUG level. 
 
+## Frame Copies
+
+The following diagram specify the frame flow in the system and indicates where and when the frame is being copied/reconstructed.
+
+
+
+![](C:\Work\Git\librealsense\doc\img\frame_lifetime.png)

```

Comment:
I don't see any diagram here

Reply:
Correct, used absolute path
Changed to relative path